pkg/executables/kubectl.go

```diff
@@ -210,14 +210,9 @@ func (k *Kubectl) LoadSecret(ctx context.Context, secretObject string, secretObj
 	return nil
 }
 
-func (k *Kubectl) ApplyKubeSpec(ctx context.Context, cluster *types.Cluster, spec string) error {
-	params := []string{"apply", "-f", spec}
-	if cluster.KubeconfigFile != "" {
-		params = append(params, "--kubeconfig", cluster.KubeconfigFile)
-	}
-	_, err := k.Execute(ctx, params...)
-	if err != nil {
-		return fmt.Errorf("executing apply: %v", err)
+func (k *Kubectl) ApplyManifest(ctx context.Context, kubeconfigPath, manifestPath string) error {
```

Comment:
nit: why the change from `spec` to `manifestPath`?

Reply:
spec is a bit ambiguous IMO
What is spec? the spec of an object? the spec of a cluster?
what this argument is is the path to a file containing a yaml manifest